apps/messagegui/app.js

```diff
@@ -24,6 +24,7 @@ require("messages").pushMessage({"t":"add","id":"call","src":"Phone","title":"Bo
 var Layout = require("Layout");
 var layout; // global var containing the layout for the currently displayed message
 var settings = require('Storage').readJSON("messages.settings.json", true) || {};
+var reply = require("reply");
```

Comment:
I'm not sure if you need:

```
var reply;
try { reply = require("reply"); } catch (e) {}
```

It might be worth trying without `reply` and seeing what happens as I think it'll throw an exception which will stop the rest of the app loading

Reply:
I hit this myself while testing yesterday and you're right, it hangs the app if reply isn't installed - will add some safety checks here!